### DISCUSSION TITLE
Anyone having issues with Thumbs.db ?

### DISCUSSION THREAD

**Discussion by turexy** [upscayl/upscayl#676] [Q&A]
Hello all,

Whenever I try to do batch upscale it fails at some point when the app hits Thumbs.db file. Whenever you delete the Thumbs.db it gets recreated unless you disable it in the registry but this have implications where whenever you open a folder with photos thumbnails have to be regenerated from scratch each time.  I think it would be useful to either have option to ignore this (or any system) file or have the app to ignore any hidden/system files by default. 

What are your thoughts?

**Comment by NayamAmarshe** [comment]
Sorry about that, the new version will ignore hidden files.

**Comment by Wyrdgirn** [comment]
That file is generated by Windows Explorer, it has information about the thumbnails that the explorer uses in the files and folders, I suppose it is a bug in the app which does not filter the extensions of the input files, I suppose... We have to wait for new updates for it to be corrected...

If it still happens, you can try transfer the images to external storage (USB, SD, etc...) and do the batch scaling from there, it is a bit annoying but it might help you maybe...

**Reply by aaronliu0130** [reply]
We should only batch upscayl image files now
